pep/app/services/head-data.js

```diff
@@ -2,5 +2,5 @@ import Service from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
 export default class HeadDataService extends Service {
-    @tracked themePath?: string;
+    @tracked themePath;
```

Comment:
Is there a reason you're removing the type here?

Reply:
its a .js file now 🙈 i converted this service from JS to TS because its overriding a js service from an addon and was giving one of those collision warnings we get. And in this case it was warranted to fix that, because its possible for a build to randomly choose the addon's version over ours left as-is.

Reply:
Oh whoops didn't notice that 😄 yeah the collisions are scary business. Good to go 👍